npm/ng-packs/packages/oauth/src/lib/services/oauth.service.ts

```diff
@@ -83,4 +83,20 @@ export class AbpOAuthService implements IAuthService {
 
     return this.oAuthService.fetchTokenUsingGrant(grantType, p, headers);
   }
+
+  getRefreshToken(): string {
+    return this.oAuthService.getRefreshToken();
+  }
+
+  getAccessToken(): string {
+    return this.oAuthService.getAccessToken();
+  }
+
+  refreshToken(): Promise<AbpAuthResponse> {
+    return this.oAuthService.refreshToken();
+  }
```

Comment:
Return type is not the same

![image](https://github.com/abpframework/abp/assets/49063256/6d0bd120-41b8-4c9d-841f-5dc2e408ada4)